### PR TITLE
Redesign and simplify container creation/removal

### PR DESCRIPTION
**Pull request by troglobit**
## Description

The main purpose of this PR is to fix the unreliable behavior seen in #835, a few other bugs were also hunted down and fixed in the process: 

 - Pager does not exit if content fits on first screen, e.g., `show containers`
 - Avahi (mDNS) spams logs ... now silenced
 - Zeroconf agent not stopped when interfaces are removed
 - Finit template fix, e.g., creating another container marks all other containers "dirty"

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [X] Bugfix
  - [ ] Regression tests
  - [X] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Test changes
  - [ ] Checked in changed Readme.adoc (make test-spec)
  - [ ] Added new test to group Readme.adoc and yaml file
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
  - [ ] ChangeLog updated (for major changes)
- [ ] Other (please describe):

### CODE REVIEW

File: src/confd/src/infix-containers.c [hunk 1]
```diff
@@ -191,55 +194,27 @@ static int add(const char *name, struct lyd_node *cif)
...
-	 * before being enqueued to execd, so we postpone it using this
-	 * "inbox" to the post hook.
-	 */
-	writesf(script, "a", "%s", _PATH_INBOX);
+	systemf("initctl -bnq touch container@%s.conf", name);
 
 	return 0;
 }
 
 static int del(const char *name)
```
Comment (wkz) [inline src/confd/src/infix-containers.c:207]: This is not necessarily new behavior in this series, but I think the fact that we treat "no enabled" in the same way as removing the container as the same state might be problematic.

AFAIK, that means we will also remove any associated volumes, which I don't think is what the user (or yours truly) expects.

**Review by wkz — CHANGES_REQUESTED** [review]
All hail @troglobit - destroyer of complexity! :laughing: 

Just want to check the "no enable" behavior before approving.

File: package/execd/tmpfiles.conf [hunk 2]
```diff
@@ -1,7 +1 @@
-d  /run/containers/args		0700 - -
-d  /run/containers/files	0700 - -
-d  /var/lib/containers/oci	0755 - -
-d  /run/containers/inbox	0700 - -
 d  /run/containers/queue	0700 - -
```
[the remark this answers is not in the document]
Comment (troglobit) [reply package/execd/tmpfiles.conf:1]: Had initially planned to drop it completely, but then decided to keep it in the tree for now.

File: src/confd/src/infix-containers.c (hunk 1 above)
Comment (troglobit) [reply src/confd/src/infix-containers.c:207]: Wow, good catch!  You are absolutely right, this is old buggy behavior.  Reworking to instead disable the container service, keeping any created container and volumes associated with it so they are not auto-pruned.

**Review by wkz — APPROVED** [review]
Perfect! :medal_sports:

**Pull request merged**